ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/JsonRpcMethodsFactory.java

```diff
@@ -322,6 +322,7 @@
           new AdminChangeLogLevel(parameter));
     }
     if (rpcApis.contains(RpcApis.EEA)) {
+      //      TODO: josh-richardson change this
```

Comment:
Why is there still a TODO? ...shouldn't this have been done before committing to master?